in_toto/model_test.go

```diff
@@ -1509,3 +1513,62 @@ func TestValidatePublicKey(t *testing.T) {
 		}
 	}
 }
+
+type nilsigner int
+
+func (n nilsigner) Sign(data []byte) ([]byte, string, error) {
+	return data, "nil", nil
+}
+
+func (n nilsigner) Verify(keyID string, data, sig []byte) (bool, error) {
+
+	if keyID == "nil" {
+		var same = len(data) == len(sig)
```

Comment:
Am I missing something here or could we just write: `if len(data) != len(sig)`?

Reply:
No, I think I initially had a more complex implementation in mind, but it was scrapped but I forgot to clean up all of it :). I'll fix.